src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+  background-color: #cecece;
+}
+
+header {
+  width: 100%;
+  box-sizing: border-box;
+  background-color: rgb(255, 255, 255);
+  max-height: 60px;
+  display: flex;
+  overflow: hidden;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  margin: 10px auto 10px 50px;
+}
+
+nav {
+  display: flex;
+  text-align: center;
+  margin: 0 50px 0 auto;
+  height: 60px;
+  align-items: center;
+}
+
+ul {
+  position: relative;
+  justify-content: center;
+  padding-left: 0;
+  margin: auto 0 auto 20px;
+  display: flex;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  padding: 23px 0;
+}
+
+a:visited {
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+}
+
+a:active {
+  color: #00acdc;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.is-active::after {
+  content: '';
+  background-color: #00acdc;
+  height: 2px;
+  width: 100%;
+  border-radius: 8px;
+  border: 1px solid #00acdc;
+  display: flex;
+  position: absolute;
+  margin-top: 19px;
```

Comment:
and if you move the height property to the link instead of nav, you wont need to set margin here